jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.60-to-7.61.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE TaskEvent ADD COLUMN currentOwner varchar2(255);
```

Comment:
incorrect syntax for alter statement. `COLUMN` is an invalid identifier
```suggestion
ALTER TABLE TaskEvent ADD currentOwner varchar2(255);
```